### DISCUSSION TITLE
邮箱收不到验证码

### DISCUSSION THREAD

**Discussion by abortabort** [getlantern/lantern-forum-cn#436] [Q&A] [answered]
问题已解决！

**Comment by MaxMKMax** [comment]
@abortabort 嗯嗯. 可能是客服那里收不到您的邮件. 没事, 我来帮您提交问题.

**Reply by abortabort** [reply]
@Mike777Khoo 好的，谢谢。另外，账号和支付单据单独给你发过邮件的（mike@getlantern.org)，也给 support@getlantern.org 发过邮件，也在软件界面提交过问题，已经近10天了，全都音讯全无。请尽快解决问题。

**Reply by MaxMKMax** [reply]
@abortabort 好的. 我今晚帮您催他们给您发验证码. 非常抱歉让您久等了.

因为我不负责客服的工作, 所以如果您给我发邮件, 最好是在Github通知我一下. 不然我没留意到. 

最好的方法是您在这里找我或直接发邮件到我们的客服. 像这种情况就在Github找我. 

真的非常抱歉啦, 团队一上线我就来帮您问他们要. 不好意思啊. 谢谢您的耐心等待!

**Reply by MaxMKMax** [reply, marked as answer]
@abortabort 明天或后天请您查看邮箱或GitHub的这个帖子. 他们在帮您弄了. 不好意思让您久等了. 谢谢您支持蓝灯!
